modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -27,96 +32,183 @@ The repository must be accessible from the hub cluster and be defined as a sourc
 +
 [source,yaml]
 ----
+---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: sriovdata
-  namespace: ztp-site
+  name: group-hardware-types-configmap
+  namespace: ztp-group
   annotations:
     argocd.argoproj.io/sync-options: Replace=true <1>
 data:
-  example-sno-du_fh-numVfs: "8"
-  example-sno-du_fh-pf: ens1f0
-  example-sno-du_fh-priority: "10"
-  example-sno-du_fh-vlan: "140"
-  example-sno-du_mh-numVfs: "8"
-  example-sno-du_mh-pf: ens3f0
-  example-sno-du_mh-priority: "10"
-  example-sno-du_mh-vlan: "150"
+  # SriovNetworkNodePolicy.yaml
+  hardware-type-1-sriov-node-policy-pfNames-1: "[\"ens5f0\"]"
+  hardware-type-1-sriov-node-policy-pfNames-2: "[\"ens7f0\"]"
+  # PerformanceProfile.yaml
+  hardware-type-1-cpu-isolated: "2-31,34-63"
+  hardware-type-1-cpu-reserved: "0-1,32-33"
+  hardware-type-1-hugepages-default: "1G"
+  hardware-type-1-hugepages-size: "1G"
+  hardware-type-1-hugepages-count: "32"
 ----
 <1> The `argocd.argoproj.io/sync-options` annotation is required only if the `ConfigMap` is larger than 1 MiB in size.
+
+.. Create `group-zones-configmap`, a `ConfigMap` to hold the regional configuration. For example:
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: group-zones-configmap
+  namespace: ztp-group
+data:
+  # ClusterLogForwarder.yaml
+  zone-1-cluster-log-fwd-outputs: "[{\"type\":\"kafka\", \"name\":\"kafka-open\", \"url\":\"tcp://10.46.55.190:9092/test\"}]"
+  zone-1-cluster-log-fwd-pipelines: "[{\"inputRefs\":[\"audit\", \"infrastructure\"], \"labels\": {\"label1\": \"test1\", \"label2\": \"test2\", \"label3\": \"test3\", \"label4\": \"test4\"}, \"name\": \"all-to-default\", \"outputRefs\": [\"kafka-open\"]}]"
+----
+
+.. Create `site-data-configmap`, a `ConfigMap` to hold the site specific configuration. For example:
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: site-data-configmap
+  namespace: ztp-group
+data:
+  # SriovNetwork.yaml
+  du-sno-1-zone-1-sriov-network-vlan-1: "140"
+  du-sno-1-zone-1-sriov-network-vlan-2: "150"
+----
 +
 [NOTE]
 ====
-The `ConfigMap` must be in the same namespace with the policy that has the hub template substitution.
+Each `ConfigMap` must be in the same namespace as the policy to be generated from the group `PolicyGenTemplate` CR that is using hub templates.
 ====
 
-. Commit the `ConfigMap` CR in Git, and then push to the Git repository being monitored by the Argo CD application.
+. Commit the `ConfigMap` CRs in Git, and then push to the Git repository being monitored by the Argo CD application.
 
-. Create a site PGT CR that uses templates to pull the required data from the `ConfigMap` object. For example:
+. Apply the hardware type and region labels to the clusters. 
+The following command applies to a single cluster named `du-sno-1-zone-1` and the labels chosen are `"hardware-type": "hardware-type-1"` and `"group-du-sno-zone": "zone-1"`:
```

Comment:
Is this correct? Wouldn't this command patch any resource with the label `managedclusters.cluster.open-cluster-management.io/du-sno-1-zone-1`?

Reply:
This command is correct. It will patch the labels for the `du-sno-1-zone-1` managed cluster.